logic/ns.go

```diff
@@ -128,6 +129,10 @@ func (qtx *Tx) CreateNamespace(ctx context.Context, ns NamespaceContent) error {
 		return err
 	}
 
+	if _, exists := ns.Properties["created_at"]; !exists {
+		ns.Properties["created_at"] = time.Now().Format(time.RFC3339)
```

Comment:
To be consistent with the rest of the Iceberg spec where timestamps are stored, can we instead store it as milliseconds since epoch. You can compute it via `time.Now().UnixMilli()`.

In the example that the Iceberg REST spec provides, that's what they do as well.
```json
{
  "namespace": [
    "accounting",
    "tax"
  ],
  "properties": {
    "owner": "Ralph",
    "created_at": "1452120468"
  }
}
```

Reply:
Since `Namespace.properties` is map of string -> string, I have used `strconv` to format the timestamp in the required format.